scipy/spatial/ckdtree.pyx

```diff
@@ -478,6 +498,10 @@ cdef public class cKDTree [object ckdtree, type ckdtree_type]:
         np.ndarray               boxsize_data
         np.float64_t             *raw_boxsize_data
 
+    property size:
```

Comment:
Make this a readonly attribute of type ```np.intp_t``` instead, otherwise we must document it as a property, which makes no sense. It also requires that we change the C++ ```struct ckdtree``` accordingly. Set the value in ```__init__```.